### ISSUE TITLE
Feature idea 

### ISSUE THREAD

**Issue by Lrnvrybthksa** [chaosprint/glicol#143]
Maybe for visualizing where the sequencer is playing 

Like showing which seq is currently playing 

Like the time meter moving through audio in a DAW 

Maybe 2 visualization windows

One for only performer to see (like maybe certain graphs or sequences to work in before showing audience? Or to browse docs or something) 

And the other to show audience? 

Just some thoughts I had, sorry if they don't make sense

**Comment by chaosprint** [comment]
Yes that's a good idea and it can be seen on other live coding platforms like Gibber or Strudel.
The main issue at the moment is that I found working with web code editor and CSS very frustrating so I am writing a code editor renderer and UI library from scratch. It will take some time but once we get that it will be much more flexible.

**Comment by Lrnvrybthksa** [comment]
> Yes that's a good idea and it can be seen on other live coding platforms like Gibber or Strudel.
> The main issue at the moment is that I found working with web code editor and CSS very frustrating so I am writing a code editor renderer and UI library from scratch. It will take some time but once we get that it will be much more flexible.

Woah that sounds intense but cool lol 😆 and that's all in rust and Js?

**Comment by chaosprint** [comment]
> > Yes that's a good idea and it can be seen on other live coding platforms like Gibber or Strudel.
> > The main issue at the moment is that I found working with web code editor and CSS very frustrating so I am writing a code editor renderer and UI library from scratch. It will take some time but once we get that it will be much more flexible.
> 
> Woah that sounds intense but cool lol 😆 and that's all in rust and Js?

basically all in rust but js bindings will be provided as well

**Comment by chaosprint** [comment]
For this idea there is actually a demo here: https://quaverseries.web.app/
Pay attention to the tab title.
Is this the style you suggest?
Or the style similar to https://strudel.cc/
